core/src/test/scala/unit/kafka/log/LogManagerTest.scala

```diff
@@ -1392,4 +1415,4 @@ object LogManagerTest {
     data.topicStates().addAll(topics.values())
     new LeaderAndIsrRequest(data, 7.toShort)
   }
-}
\ No newline at end of file
+}
```

Comment:
ditto

Reply:
I enable file ends with a line break in my editor. Is this preferred style in kafka? If not, I can revert it. Thanks.

Reply:
I prefer to exclude unrelated changes from PR :)